docs/griptape-framework/structures/config.md

```diff
@@ -34,6 +34,18 @@ agent = Agent(
 )
 ```
 
+#### Google
+The [Google Structure Config](../../reference/griptape/config/structure_config.md#griptape.config.structure_config.GoogleStructureConfig) provides default Drivers for Google's Gemini APIs.
```

Comment:
This link should be `../../reference/griptape/config/google_structure_config/` or `../../reference/griptape/config/google_structure_config.md`